### DISCUSSION TITLE
Welcome to Sidekick Discussions!

### DISCUSSION THREAD

**Discussion by MightyMoud** [MightyMoud/sidekick#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rubiojr** [comment]
:wave:! Sergio here. Working on this site as a soft-eng during the day. Open Source writer from time to time (when time permits). I came across Sidekick after watching [Rails World 2024 opening keynote](https://www.youtube.com/watch?v=-cEn_83zRFw) a few days ago, while searching for some simple, self-hosted PaaS in Go, running away from the complexities (or should I say horrors :thinking:) of day to day Kubernetes deployments for simple apps.

Loving it so far! :heart:.
